src/payform.coffee

```diff
@@ -209,11 +209,20 @@
 
   reFormatCardNumber = (e) ->
     return if e.target.value is ""
+    if document.dir == 'ltr'
```

Comment:
`dir` may be also set via CSS, and/or on a particular element as it doesn't have to be the same for the whole document. Consider this instead:

```coffeescript
dir = getComputedStyle(e.target)['direction']
```

Reply:
Thoughts on this @ddayguerrero ?

Reply:
I had raised this concern when I first started introducing these conditionals for document directionality. 

I will take a deeper look on this, but it seems like a viable solution and also adds more flexibility!

https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle (Support for IE9+!!!)